pyrsa/vis/rdm_plot.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 12 16:21:53 2020
+
+@author: heiko
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from pyrsa.rdm import rank_transform
+
+
+def show_rdm(rdm, do_rank_transform=False, pattern_descriptor=None,
+             cmap=None, rdm_descriptor=None):
+    """shows an rdm object
+
+    Parameters
+    ----------
+    rdm : pyrsa.rdm.RDMs
+        RDMs object to be plotted
+    do_rank_transform : bool
+        whether we should do a rank transform before plotting
+    pattern_descriptor : String
+        name of a pattern descriptor which will be used as an axis label
+    pattern_descriptor : String
+        name of a rdm descriptor which will be used as a title per RDM
+    cmap : color map
+        colormap or identifier for a colormap to be used
+        conventions as for matplotlib colormaps
+
+    """
+    if do_rank_transform:
+        rdm = rank_transform(rdm)
+    rdm_mat = rdm.get_matrices()
+    if rdm.n_rdm  > 1:
+        m = np.ceil(np.sqrt(rdm.n_rdm+1))
+        n = np.ceil((1 + rdm.n_rdm) / m)
+        for idx in range(rdm.n_rdm):
+            plt.subplot(n, m, idx + 1)
```

Comment:
So I think we're eventually going to want to have some way for the user to pass in the image context. For example when they want to include the RDM in a subplot of a figure they are making. One way would be to have an optional `axes` argument. But this may be outside of the scope of this PR.